src/gs-updates-page.c

```diff
@@ -610,6 +610,7 @@ _reboot_failed_cb (GObject *source, GAsyncResult *res, gpointer user_data)
 	GsUpdatesPage *self = GS_UPDATES_PAGE (user_data);
 	g_autoptr(GError) error = NULL;
 	g_autoptr(GsAppList) apps = NULL;
+	g_autoptr(GsApp) app = NULL;
```

Comment:
I don't think this should be `g_autoptr` since `gs_app_list_index()` does not return a new reference